README.md

```diff
@@ -1,2 +1,112 @@
 # CombineExternalAccessory
-Combine Wrapper for ExternalAccessory.framework
+Combine Wrapper for ExternalAccessory.framework.
+
+Currently implements browsing for unconfigured WAC (Wireless Accessory Configuration) devices
+and allows to start the configuration process for these in your app.
+
+This is implemented in Apple's ExternalAccessory framework. That framework (and this wrapper)
+is basically just an interface to Apple's MFi process.
+
+## Installation
+
+Installation via SPM.
+
+## Usage
+
+To get an array of the currently unconfigured devices, use and subscribe to `UnconfiguredExternalAccessoryBrowser`.
+Subscribing to this publisher will start searching for unconfigured devices and deliver these as events. 
+
+These are the possible events:
+```
+public enum Event {
+    case didFindUnconfiguredAccessories(accessories: [UnconfiguredExternalAccessory])
+    case didRemoveUnconfiguredAccessories(accessories: [UnconfiguredExternalAccessory])
+}
+```
+
+```
+UnconfiguredExternalAccessoryBrowser()
+.sink { event in
+    switch event {
+    case let .didFindUnconfiguredAccessories(accessories):
+        // Add to your list
+    case let .didRemoveUnconfiguredAccessories(accessories):
+        // Remove from your list
+    }
+}
+```
+
+Typically, you want to update your View or Store with the complete list, you could achieve this
+with e.g. `scan`. Scan is like reduce but for Combine Publishers.
+
+Example:
+```
+UnconfiguredExternalAccessoryBrowser()
+    .scan(Set<UnconfiguredExternalAccessory>(), { (aggr, event) in
```

Comment:
I would be careful with using Set here. Although it's convenient to avoid duplicated hashable elements, it loses the order and makes the UI part really tricky, it's a very easy way to crash SwiftUI (I learned that the worst way, with Dictionaries).
We can convert to List and sort in the next node, but usually for small sets of data I usually prefer working with List directly.

Reply:
The ExternalAccessory framework returns a Set here. Which is the reason I kept it here, as this publisher is supposed to be just a wrapper. Usability improvements can come later.

But I do not understand the point about Sets/Dictionaries and crashes in general.

Reply:
SwiftUI crashes sometimes when a Set/Dictionary is used in a ForEach. I faced this two or three times and it's always annoying, maybe it's solved on iOS 14 as it doesn't use UITableView anymore, but I would be careful with that.